### PR TITLE
[LTR] Implement Mordor Trebuchet

### PR DESCRIPTION
**Pull request by Susucre**
Along with its unique token Ballistic Boulder.

### CODE REVIEW

**Comment by Susucre** [comment]
[[Mordor Trebuchet]]

File: Mage.Sets/src/mage/cards/m/MordorTrebuchet.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        return new MordorTrebuchetEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Token token = new BallisticBoulder();
+        token.putOntoBattlefield(1, game, source, source.getControllerId(), true, true);
+        game.addDelayedTriggeredAbility(new AtTheEndOfCombatDelayedTriggeredAbility(
+            new SacrificeTargetEffect().setText("sacrifice it")
+                .setTargetPointer(new FixedTargets(token, game))
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/m/MordorTrebuchet.java:98]: This isn't sufficient for setting targets. Reference Phantom Steed

**Review by xenohedron — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/m/MordorTrebuchet.java (hunk 1 above)
Comment (Susucre) [reply Mage.Sets/src/mage/cards/m/MordorTrebuchet.java:98]: I do think I copied Urabrask' Forge on this one. Is that one having an issue also?
I'll give it a proper look this evening (in a few hours).

Comment (xenohedron) [reply Mage.Sets/src/mage/cards/m/MordorTrebuchet.java:98]: Maybe I'm wrong. If it works properly with Doubling Season and with populate (e.g. Eyes in the Skies) then it's good.

Comment (Susucre) [reply Mage.Sets/src/mage/cards/m/MordorTrebuchet.java:98]: So I did try it, and Doubling season has the additional token also sacrificed at end of combat, and a Populated version is not sacrificed. Same for Urabrask's Forge.
I do not see anything wrong with that.

**Review by xenohedron — APPROVED** [review]
Okay, sorry for the trouble. Thanks for confirming.

**Pull request merged**